### PR TITLE
Create platform and permissions neutral path for blog

### PR DESCRIPTION
**Pull request by JeremyLikness**
Used local folder for consistency.

### CODE REVIEW

**Comment by JeremyLikness** [comment]
Fixes #3285

File: samples/core/GetStarted/Model.cs [hunk 1]
```diff
@@ -8,10 +8,19 @@ public class BloggingContext : DbContext
         public DbSet<Blog> Blogs { get; set; }
         public DbSet<Post> Posts { get; set; }
 
+        public string DbPath { get; private set; }
+
+        public BloggingContext()
+        {
+            var folder = System.Environment.SpecialFolder.LocalApplicationData;
+            var path = System.Environment.GetFolderPath(folder);
+            DbPath = $"{path}\\blogging.db";
```
Comment (ErikEJ) [inline samples/core/GetStarted/Model.cs:17]: Should it not be forward slash (or Env.Pathseparatir) ?

Comment (JeremyLikness) [reply samples/core/GetStarted/Model.cs]: Great catch! Submitted fix.

**Comment by ErikEJ** [comment]
Should the comment in line 21 change now?

**Comment by ErikEJ** [comment]
LGTM 😁

**Comment by JeremyLikness** [comment]
> LGTM 😁

Whew!

**Review by AndriySvyryd — APPROVED** [review]

**Pull request merged**